keyboards/qwertykeys/qk80/ansi/ansi.c

```diff
@@ -0,0 +1,174 @@
+    {0, CS5_SW3, CS6_SW3, CS4_SW3},    /* RGB35-Q */
+    {0, CS8_SW3, CS9_SW3, CS7_SW3},    /* RGB36-W */
+    {0, CS11_SW3, CS12_SW3, CS10_SW3}, /* RGB37-E */
+    {0, CS14_SW3, CS15_SW3, CS13_SW3}, /* RGB38-R */
+    {0, CS17_SW3, CS18_SW3, CS16_SW3}, /* RGB39-T */
+    {0, CS20_SW3, CS21_SW3, CS19_SW3}, /* RGB40-Y */
+    {0, CS23_SW3, CS24_SW3, CS22_SW3}, /* RGB41-U */
+    {0, CS26_SW3, CS27_SW3, CS25_SW3}, /* RGB42-I */
+    {0, CS29_SW3, CS30_SW3, CS28_SW3}, /* RGB43-O */
+    {0, CS32_SW3, CS33_SW3, CS31_SW3}, /* RGB44-P */
+    {0, CS35_SW3, CS36_SW3, CS34_SW3}, /* RGB45-[ */
+    {0, CS38_SW3, CS39_SW3, CS37_SW3}, /* RGB46-] */
+    {0, CS35_SW9, CS36_SW9, CS34_SW9}, /* RGB47-\ */
+    {0, CS26_SW8, CS27_SW8, CS25_SW8}, /* RGB48-Delete */
+    {0, CS29_SW8, CS30_SW8, CS28_SW8}, /* RGB49-End */
+    {0, CS32_SW6, CS33_SW6, CS31_SW6}, /* RGB50-PageDown */
+    {0, CS2_SW4, CS3_SW4, CS1_SW4},    /* RGB51-CAPS---ROW3*/
+    {0, CS5_SW4, CS6_SW4, CS4_SW4},    /* RGB52-A-- */
+    {0, CS8_SW4, CS9_SW4, CS7_SW4},    /* RGB53-S-- */
+    {0, CS11_SW4, CS12_SW4, CS10_SW4}, /* RGB54-D */
+    {0, CS14_SW4, CS15_SW4, CS13_SW4}, /* RGB55-F */
+    {0, CS17_SW4, CS18_SW4, CS16_SW4}, /* RGB56-G */
+    {0, CS20_SW4, CS21_SW4, CS19_SW4}, /* RGB57-H */
+    {0, CS23_SW4, CS24_SW4, CS22_SW4}, /* RGB58-J */
+    {0, CS26_SW4, CS27_SW4, CS25_SW4}, /* RGB59-K */
+    {0, CS29_SW4, CS30_SW4, CS28_SW4}, /* RGB60-L */
+    {0, CS32_SW4, CS33_SW4, CS31_SW4}, /* RGB61-;: */
+    {0, CS35_SW4, CS36_SW4, CS34_SW4}, /* RGB62-'" */
+    {0, CS38_SW4, CS39_SW4, CS37_SW4}, /* RGB63-ENTER */
+    {0, CS2_SW5, CS3_SW5, CS1_SW5},    /* RGB64-LSF --ROW4*/
+    {0, CS5_SW5, CS6_SW5, CS4_SW5},    /* RGB65-Z */
+    {0, CS8_SW5, CS9_SW5, CS7_SW5},    /* RGB66-X */
+    {0, CS11_SW5, CS12_SW5, CS10_SW5}, /* RGB67-C */
+    {0, CS14_SW5, CS15_SW5, CS13_SW5}, /* RGB68-V */
+    {0, CS17_SW5, CS18_SW5, CS16_SW5}, /* RGB69-B */
+    {0, CS20_SW5, CS21_SW5, CS19_SW5}, /* RGB70-N */
+    {0, CS23_SW5, CS24_SW5, CS22_SW5}, /* RGB71-M */
+    {0, CS26_SW5, CS27_SW5, CS25_SW5}, /* RGB72-,< */
+    {0, CS29_SW5, CS30_SW5, CS28_SW5}, /* RGB73->. */
+    {0, CS32_SW5, CS33_SW5, CS31_SW5}, /* RGB74-?/ */
+    {0, CS35_SW5, CS36_SW5, CS34_SW5}, /* RGB75-RSF */
+    {0, CS35_SW6, CS36_SW6, CS34_SW6}, /* RGB76-UP */
+    {0, CS2_SW6, CS3_SW6, CS1_SW6},    /* RGB77-lct-- ROW5*/
+    {0, CS5_SW6, CS6_SW6, CS4_SW6},    /* RGB78-lwin */
+    {0, CS8_SW6, CS9_SW6, CS7_SW6},    /* RGB79-lalt */
+    {0, CS14_SW6, CS15_SW6, CS13_SW6}, /* RGB80-sp */
+    {0, CS23_SW6, CS24_SW6, CS22_SW6}, /* RGB81-rwin */
+    {0, CS26_SW6, CS27_SW6, CS25_SW6}, /* RGB82-FN */
+    {0, CS29_SW6, CS30_SW6, CS28_SW6}, /* RGB83-rctL */
+    {0, CS38_SW6, CS39_SW6, CS37_SW6}, /* RGB84-left */
+    {0, CS26_SW9, CS27_SW9, CS25_SW9}, /* RGB85-down */
+    {0, CS29_SW9, CS30_SW9, CS28_SW9}  /* RGB86-right */
+};
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
```

Comment:
Since you have two boards that are using this code, it may make more sense to move this to qk80.c in the qk80 folder to reduce redundant code.